content/en/docs/concepts/services-networking/network-policies.md

```diff
@@ -312,6 +322,58 @@ The value of the label is the namespace name.
 While NetworkPolicy cannot target a namespace by its name with some object field, you can use the
 standardized label to target a specific namespace.
 
+## Pod lifecycle
+
+When the NetworkPolicy rules should be applied in the pod's lifecycle is undefined, since
+there is a potential race if the pod and the NetworkPolicy that selects that pod are created
+at around the same time. Nevertheless, if the policy was created a sufficiently long time
```

Comment:
Why there needs a "sufficiently long time"?
How do we define "sufficiently long"?

Reply:
I removed that part, and converted it to the less-detailed "initial intent" explanation